weldx/asdf/schemas/weldx.bam.de/weldx/core/graph/signal-1.0.0.yaml

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/schemas/weldx/core/graph/signal-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/graph/signal-1.0.0"
+
+title: |
+  signal_transform
+
+type: object
+properties:
+  name:
+    type: string
+  value:
+    type: number
+
+required: [name, value]
```

Comment:
Still not sure if `name` should be a requirement in the schema of a signal

Reply:
I agree, the measurement chain examples are just dummy code here